www/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -21,102 +21,146 @@ import StickyResponsiveSidebar from 'components/StickyResponsiveSidebar';
 import TitleAndMetaTags from 'components/TitleAndMetaTags';
 import findSectionForPath from 'utils/findSectionForPath';
 import toCommaSeparatedList from 'utils/toCommaSeparatedList';
-import {sharedStyles} from 'theme';
+import {media, sharedStyles} from 'theme';
 import {urlRoot} from 'constants';
 
-const MarkdownPage = ({
-  authors,
-  date,
-  ogDescription,
-  location,
-  markdownRemark,
-  sectionList,
-  titlePostfix = '',
-}) => {
-  const hasAuthors = authors.length > 0;
-  const titlePrefix = markdownRemark.frontmatter.title || '';
+class MarkdownPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggleMenuOverlay = this.toggleMenuOverlay.bind(this);
+    this.handleWindowResize = this.handleWindowResize.bind(this);
+    this.state = {
+      isMenuOverlayOpen: false,
+    };
+  }
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.isMenuOverlayOpen && !prevState.isMenuOverlayOpen) {
+      // Menu has been opened
+      window.addEventListener('resize', this.handleWindowResize);
+    }
+  }
+  handleWindowResize() {
+    const mediaQuery = media.greaterThan('small').replace('@media ', '');
+    if (window.matchMedia(mediaQuery).matches) {
+      // If the screen has been resized >= 'small'
+      this.toggleMenuOverlay(false);
+    }
+  }
```

Comment:
super-nit: I usually put newlines between method definitions. Not sure we are doing that in the docs code though.

Reply:
Good catch, I'll do that now